docs/source/notebooks/sampling_conjugate_step.ipynb

```diff
@@ -0,0 +1,634 @@
+{
```

Comment:
<ul><li>Move the comment in the <code>if</code> statement instead of the <code>else</code> ﻿</li><li>To remove the <code>FutureWarning</code>, just set <code>return_inferencedata=True</code> in <code>pm.sample</code>, and then call <code>az.summary</code> directly instead of the <code>pm</code> alias</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/4199/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>